docs/src/components/ComponentSandpack.jsx

```diff
@@ -19,7 +19,7 @@ export default function ComponentSandpack({
         }}
         customSetup={{
           dependencies: {
-            'media-chrome': 'latest',
+            'media-chrome': 'canary',
```

Comment:
So that `defaultsubtitles` on media-controller works, we want to use the canary of media-chrome.
Once media-chrome 1.0 is released, we'd want to consider switching this back to `latest`.